frontend/appflowy_tauri/src/appflowy_app/components/document/BlockSideTools/BlockSideTools.hooks.tsx

```diff
@@ -53,12 +51,16 @@ export function useBlockSideTools({ container }: { container: HTMLDivElement })
         }
       }
     }
-  }, [nodeId, nodes]);
+  }, [nodeId]);
 
-  const handleAddClick = useCallback(() => {
-    if (!nodeId) return;
-    insertAfter(nodes[nodeId]);
-  }, [nodeId, nodes]);
+  const handleOpenMenu = useCallback(() => {
+    setMenuOpen(true);
+  }, []);
+
+  const handleCloseMenu = useCallback(() => {
+    setMenuOpen(false);
+    setHoverNodeId('');
+  }, []);
```

Comment:
Couldn't these be consolidated into just one callback?

Reply:
Did you mean toggle?

Reply:
Yeah